lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -486,6 +486,22 @@ class Simulator {
     };
   }
 
+  /**
+   * @param {number} wastedBytes
+   */
+  simulateTimespan(wastedBytes) {
+    const {throughput, observedThroughput} = this._options;
+
+    // https://github.com/GoogleChrome/lighthouse/pull/13323#issuecomment-962031709
+    // 0 throughput means the throttling is unset. Use the observed throughput in this case.
+    const bitsPerSecond = throughput || observedThroughput;
```

Comment:
throughput is definitely a number and we only care about the 0 case so...even though its more code i'd rather be explicit with a `=== 0` conditional.